### ISSUE TITLE
Not possible to have static web resources along with filtered resources

### ISSUE THREAD

**Issue by shaileshaag** [spring-projects/spring-boot#2450]
Hi,
I'm trying to build a spring boot project with Swagger UI. The swagger UI static contents are placed under "src/main/static". There are few properties files which are placed under "src/main/resources". The property files under resources folder have place-holders which I need to be pre-filled. At the same time, there is no such filter requirements under "static" folder.
I've configured the "spring-boot-maven-plugin" with "<addResources>false</addResources>".

When starting the app using:
mvn spring-boot:run

It is not picking up the resources under "static" folder.

So, in a nutshell, I need to configure project with partially "filtered" resources and a few static resources which do not have filtering requirements.

At least, the changes should be able to read / load resources specified in "maven-war-plugin" configuration (if available).

**Comment by wilkinsona** [comment]
It's not clear to me that this is a problem with Spring Boot. Perhaps you can explain in a bit more detail? Note that you can use includes and excludes to control what is and is not filtered by Maven's resource plugin. See [its documentation](http://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html) for more details.

**Comment by hoserdude** [comment]
Easiest thing to do is put your swagger UI under `src/main/resources/static`.  Your properties files can sit at the root of the resources directory.  You can configure the maven resource plugin to include or exclude whatever you want. 

```
    <build>
        <resources>
            <resource>
                <directory>${project.basedir}/src/main/resources</directory>
                <filtering>true</filtering>
                <includes>
                    <include>**/my.properties</include>
                </includes>
            </resource>
            <resource>
                <directory>${project.basedir}/src/main/resources</directory>
                <filtering>false</filtering>
            </resource>
        </resources>
```

**Comment by shaileshaag** [comment]
Thanks @hoserdude . That helped.
